static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx

```diff
@@ -123,12 +125,13 @@ type IssueListProps = {
 };
 
 export function IssueList({issues, node, organization}: IssueListProps) {
+  const panelRef = useRef<HTMLDivElement>(null);
   if (!issues.length) {
     return null;
   }
 
   return (
-    <StyledPanel>
+    <StyledPanel ref={panelRef} width={panelRef.current?.clientWidth}>
```

Comment:
Can you just remove this now? We dont need it anymore

Reply:
Woops yup, that shouldn't be there anymore.